Library/Formula/jdnssec-tools.rb

```diff
@@ -0,0 +1,21 @@
+class JdnssecTools < Formula
+  desc "Java command-line tools for DNSSEC"
+  homepage "http://www.verisignlabs.com/jdnssec-tools/"
+  url "http://www.verisignlabs.com/dnssec-tools/packages/jdnssec-tools-0.12.tar.gz"
+  sha256 "3fa0bfe062051d9048c5186adbb04d8f5e176e50da3cdcf9b0d97e1c558efc22"
+
+  depends_on :java
+
+  def install
+    inreplace Dir["bin/*"] do |s|
+      s.gsub! /basedir=.*/, "basedir=#{libexec}"
+    end
+    bin.install Dir["bin/*"]
+    (libexec/"lib").install Dir["lib/*"]
+  end
+
+  test do
+    (testpath/"powerdns.com.key").write("powerdns.com.   10773 IN  DNSKEY  257 3 8 AwEAAb/+pXOZWYQ8mv9WM5dFva8WU9jcIUdDuEjldbyfnkQ/xlrJC5zA EfhYhrea3SmIPmMTDimLqbh3/4SMTNPTUF+9+U1vpNfIRTFadqsmuU9F ddz3JqCcYwEpWbReg6DJOeyu+9oBoIQkPxFyLtIXEPGlQzrynKubn04C x83I6NfzDTraJT3jLHKeW5PVc1ifqKzHz5TXdHHTA7NkJAa0sPcZCoNE 1LpnJI/wcUpRUiuQhoLFeT1E432GuPuZ7y+agElGj0NnBxEgnHrhrnZW UbULpRa/il+Cr5Taj988HqX9Xdm6FjcP4Lbuds/44U7U8du224Q8jTrZ 57Yvj4VDQKc=")
```

Comment:
Can we break this onto multiple lines, or shorten it? It's currently making the formula impossible to read in a glance.